packages/eslint-plugin-widen/test/jsx-import.spec.js

```diff
@@ -1,14 +1,18 @@
+import tsParser from '@typescript-eslint/parser'
 import { RuleTester } from 'eslint'
 import heredoc from 'tsheredoc'
 import rule from '../src/rules/jsx-import'
 
 RuleTester.setDefaultConfig({
-  parserOptions: {
-    ecmaFeatures: {
-      jsx: true,
+  languageOptions: {
+    parser: tsParser,
+    parserOptions: {
+      ecmaFeatures: {
+        jsx: true,
+      },
+      ecmaVersion: 2018,
```

Comment:
Do we need to specify this anymore? I believe it defaults to latest and wondering if it should be. I don't remember why we were on 2018 unless it's for backwards compatibility with something... but then we should have added a note if that were the case?

Reply:
Removed as its probably fine on the default (latest)